examples/model-viewer/shaders.slang

```diff
@@ -85,7 +85,13 @@ struct BlinnPhong : IBRDF
         float3 H = normalize(L + V);
         float nDotH = saturate(dot(N, H));
 
-        return kd*nDotL + ks*pow(nDotH, specularity);
+        // TODO: The current model loading has a bug that is leading
```

Comment:
Is this still true?

Reply:
Yes. I didn't try to fix the model loading bug as part of this change. If you set a breakpoint at the places where `SimpleMaterial` instances are being created in C++ you will find that one of them is being filled in with garbage data.